integration/db/src/main/java/com/hbc/pms/integration/db/entity/AlarmHistoryEntity.java

```diff
@@ -0,0 +1,41 @@
+package com.hbc.pms.integration.db.entity;
+
+import com.hbc.pms.core.model.enums.AlarmStatusEnum;
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.OffsetDateTime;
+
+@Entity
+@Table(name = "alarm_history")
+@Builder
+@Getter
+@Setter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+public class AlarmHistoryEntity {

```

Comment:
Please add a database diagram for this entity; if it already exists, then attach it to the description of the PR.